javascript-loops-practice/get-student-names.js

```diff
@@ -0,0 +1,10 @@
+/* exported getStudentNames */
+function getStudentNames(students) {
+  var nameArray = [];
+  for (var studentsIndex = 0; studentsIndex < students.length; studentsIndex++) {
+    for (var name in students[studentsIndex]) {
+      nameArray.push(students[studentsIndex][name]);
+    }
```

Comment:
Why the loop? You only need one property off of the object, not every single property. I would just get the value of the `name` property off of the object and remove the `for...in` loop